examples/client.rs

```diff
@@ -49,52 +49,51 @@ impl Future for Process {
 pub fn main() {
     let _ = env_logger::init();
 
-    let mut core = reactor::Core::new().unwrap();;
+    let mut core = reactor::Core::new().unwrap();
     let handle = core.handle();
 
-    let tcp = TcpStream::connect(
-        &"127.0.0.1:5928".parse().unwrap(),
-        &handle);
+    let tcp = TcpStream::connect(&"127.0.0.1:5928".parse().unwrap(), &handle);
 
     let tcp = tcp.then(|res| {
         let tcp = io_dump::Dump::to_stdout(res.unwrap());
         Client::handshake(tcp)
-    })
-    .then(|res| {
-        let mut client = res.unwrap();
+    }).then(|res| {
+            let mut client = res.unwrap();
```

Comment:
8 space indent here?

Reply:
It lines it up w/ the pipe, which is really weird. I don't like it :(

Reply:
yeah this seems like a bug